tools/bridge/bridge/transfer_recorder.py

```diff
@@ -33,6 +39,28 @@ def __init__(self, minimum_balance: int) -> None:
 
         self.is_validator: Optional[bool] = None
         self.balance: Optional[int] = None
+        self.last_fetcher_reached_head_event: Dict[
+            ChainRole, FetcherReachedHeadEvent
+        ] = {}
+
+    def get_state_summary(self):
+        return {
+            "is-validator": self.is_validator,
+            "balance": str(self.balance or -1),
+            "num-transfer-events": len(self.transfer_events),
```

Comment:
maybe also include the transfer events directly, could be helpful to track a specific transfer

Reply:
All of them? I probably wouldn't do it in this endpoint, because that could be many transfers and it makes looking at the output a bit harder.
Though we could have another one providing a complete list.